apps/modernization-ui/src/apps/search/patient/PatientCriteria/Address.tsx

```diff
@@ -0,0 +1,83 @@
+import { Input } from 'components/FormInputs/Input';
+import { SelectInput } from 'components/FormInputs/SelectInput';
+import { Controller, useFormContext } from 'react-hook-form';
+import { SearchCriteriaContext, SearchCriteriaProvider } from 'providers/SearchCriteriaContext';
+import styles from './address.module.scss';
+
+export const Address = () => {
+    const { control } = useFormContext();
```

Comment:
Suggestion: You can type the form by declaring invoking useFormContext like this: `useFormContext<CreateValuesetRequest>()`

This gives you type checking on your Controller's so that the `name` field's will autocomplete and show a warning of you have an invalid name.